models/info.go

```diff
@@ -131,3 +131,11 @@ type StatisticsList struct {
 
 	Pagination
 }
+
+type SlashCommand struct {
+	Command         string `json:"command"`
+	Params          string `json:"params,omitempty"`
+	Description     string `json:"description,omitempty"`
+	ClientOnly      bool   `json:"clientOnly"`
+	ProvidesPreview bool   `json:"providesPreview"`
+}
```

Comment:
Can you move this to its own file in models folder?

Reply:
you mean to make new file for slash commands ?

Reply:
@geekgonecrazy Just want to confirm should I make new files for slash commands or make a file which will contain some common server functions ?

Reply:
yes exactly a new file like: models/slashCommand.go with this in it.  Instead of putting inside of models/info.go